src/main/java/util/FileUtil.java

```diff
@@ -0,0 +1,25 @@
+package util;
+
+import org.testng.ITestResult;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class FileUtil {
+
+    /***
+     * Clear log file
+     */
+    public static void clearFile(String file) throws IOException {
+        File logFile = new File(file);
+        new FileWriter(logFile, false).close();
+    }
+
+    /***
+     * Autodelete of generated video
+     */
+    public static void deleteFolderWithVideo(ITestResult result, String extension) {
```

Comment:
This method is not reusable. Are you going to create the same method with a different name If you need to delete a different folder?

Reply:
fixed